lib/cli.rb

```diff
@@ -57,11 +58,17 @@ class Cli
 
   GLI::Commands::Help.skips_post = false
 
-  def self.check_exceeding_arguments(defined, parsed)
-    if parsed.size > defined.size
+  def self.validate_number_of_arguments(defined, parsed)
```

Comment:
Assignment Branch Condition size for validate_number_of_arguments is too high. [18.92/15]